taxi/views.py

```diff
@@ -62,18 +64,39 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
     model = Car
 
 
+def car_driver_assign_or_delete(request, pk):
+    if request.method == "POST":
+        driver_id = request.POST["driver_id"]
+        car = Car.objects.get(id=pk)
+        remove = request.POST.get("remove")
```

Comment:
Maybe better just to check if the driver inside car.drivers? Remove redundant variable